mslice/util/mantid.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import (absolute_import, division, print_function)
+
+import mantid.simpleapi as s_api
+from mantid.api import AlgorithmFactory
+from mslice.models.workspacemanager.workspace_provider import add_workspace
+from mslice.models.projection.powder.make_projection import MakeProjection
+from mslice.workspace import wrap_workspace
+from mslice.workspace.base import WorkspaceBase as Workspace
+
+
+def initialize_mantid():
+    AlgorithmFactory.subscribe(MakeProjection)
+    s_api._create_algorithm_function('MakeProjection', 1, MakeProjection())
+
+
+def run_alg(alg_name, output_name=None, store=True, **kwargs):
+    return run_algorithm(getattr(s_api, alg_name), output_name, store, **kwargs)
+
+def run_algorithm(algorithm, output_name=None, store=True, **kwargs):
```

Comment:
I think the `algorithm` parameter might be better named as `algorithm_function`.